src/core/ProgressBar/__tests__/ProgressBar.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { KuberaThemeProvider } from '../../../theme';
+import { ProgressBar } from '../ProgressBar';
+
+describe('ProgressBar', () => {
+  it('Renders', () => {
+    const { getAllByRole } = render(
+      <KuberaThemeProvider platform="kubera-chaos">
+        <ProgressBar value={80} label="Success" color={'red'} />
+      </KuberaThemeProvider>
+    );
+    expect(getAllByRole('progressbar')).toBeTruthy();
```

Comment:
can we make the test case more extensive checking if the value is shown correctly or the color is correct or the label is present or not?

Reply:
@arkajyotiMukherjee Not sure, need to check it